src/Samples/FileDialogs.CmdMsg/Program.fs

```diff
@@ -113,17 +116,19 @@ open Core
 open Platform
 
 
+let vm = ViewModel.designInstance (initModel ()) (bindings ())
```

Comment:
How do you feel about renaming this (and others) to `designVm`, just to be entirely clear?

Reply:
Sure.  Good suggestion.